distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -989,9 +1003,9 @@ type Column
         result = case self.value_type of
             Value_Type.Char _ _ -> self.is_empty
             Value_Type.Float _ ->
-                if treat_nans_as_blank then self.is_nothing.iif True (self.internal_is_nan on_missing=False) else self.is_nothing
+                if treat_nans_as_blank then self.is_nothing.iif True self.internal_is_nan else self.is_nothing
```

Comment:
Is this computing `self.is_nothing` twice?

Reply:
Good observation!! Yes, apparently it is.
Well, if it's a `LongStorage`, then it is a O(1) operation, because it just re-uses the bitset used to mark missing values. But I guess better to cache it just to be sure.